bids/layout/layout.py

```diff
@@ -856,7 +857,8 @@ def get_metadata(self, path, include_entities=False, scope='all'):
             files, the values in files closer to the input filename will take
             precedence, per the inheritance rules in the BIDS specification.
         """
-
+        if path and not op.isabs(path):
+            path = op.join(self.root, path)
```

Comment:
I don't think you need the conditional; aside from the fact that `path` is mandatory, `os.path.join` will always return the last argument without modification if it's an absolute path. So just `op.join(self.root, path)` is probably fine.